pkg/cmd/codespace/ssh.go

```diff
@@ -37,14 +37,15 @@ const automaticPrivateKeyName = "codespaces.auto"
 var errKeyFileNotFound = errors.New("SSH key file does not exist")
 
 type sshOptions struct {
-	codespace  string
-	profile    string
-	serverPort int
-	debug      bool
-	debugFile  string
-	stdio      bool
-	config     bool
-	scpArgs    []string // scp arguments, for 'cs cp' (nil for 'cs ssh')
+	codespace         string
+	profile           string
+	serverPort        int
+	debug             bool
+	debugFile         string
+	stdio             bool
+	config            bool
+	codespacesKeyPair bool
```

Comment:
nit: I was thinking something like autoGenerateKeys etc.

Reply:
That was actually what I started with :) 

The reason I changed was because the flag doesn't always cause the `ssh` command to _generate_ the key pair. It will use an existing pair if one exists. Users might think that providing this flag will always wipe out the existing keys to generate a new pair with the "generate" verbiage.